depmapomics/qc/test_compare_to_ref_release.py

```diff
@@ -290,17 +290,3 @@ def test_mutation_legacy_data(data):
 #     data1_ = data1.loc[row, col].T
 #     data2_ = data2.loc[row, col].T
 #     assert_frame_equal(data1_, data2_, rtol=rtol)
-
-PARAMS_sharedcellines_are_equal = [((x['file'], x['merge_cols']), x['merge_cols'])
```

Comment:
is this change ok? if so we can merge dev into master

Reply:
@javadnoorb

Reply:
yes, I think I added a bunch of more specific tests that covered this. So I dropped it. Thanks for checking.